backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/VersionInputHandler.java

```diff
@@ -381,4 +386,63 @@ public VersionInputHandler(
         },
         executor);
   }
+
+  public InternalFuture<Map<String, VersioningEntry>> getVersionedInputs(Set<String> runIds) {
```

Comment:
Same comment to add documentation here. The map and the last compose are very not clear to me

Reply:
Try to explain the code via adding comment in code

Reply:
This method doesn't seem to be called by anything.

Reply:
it was there in commit https://github.com/VertaAI/modeldb/pull/2206/commits/bdbd5f02fd05ae328b25168cb298d29da17a54cd but due to merge conflicts it was removed. Now added removed code